src/services/codefixes/importFixes.ts

```diff
@@ -317,7 +321,7 @@ namespace ts.codefix {
     ): string[] {
         const { baseUrl, paths, rootDirs } = options;
         const choicesForEachExportingModule = mapIterator(arrayIterator(moduleSymbols), moduleSymbol => {
-            const moduleFileName = moduleSymbol.valueDeclaration.getSourceFile().fileName;
+            const moduleFileName = getOriginalModulePath(program, moduleSymbol.valueDeclaration.getSourceFile());

```

Comment:
Also this doesnt seem right... If the sourceFile2(from firstDefined call from getOriginalModulePath) which included this moduleSymbol is in different folder than the sourceFile this fix is asked for it might not be correct right? The sourceFile could refer to the symlinked location through some other node_modules symlink... (in the sense orignalFilePath for referencing this moduleSymbol could be different while resolving through sourceFile and sourceFile2?

Reply:
It's possible that there are multiple symlinks to the same module, although it's not technically *wrong* to use one or the other since it works either way; we could iterate over all known symlinks though and take the best, would that be better?